documentation/domains/Domain.md

```diff
@@ -17,6 +17,7 @@ The specification of the operation of the WebLogic domain. Required.
 | `adminServer` | [Admin Server](#admin-server) | Lifecycle options for the Administration Server, including Java options, environment variables, additional Pod content, and which channels or network access points should be exposed using a NodePort Service. |
 | `allowReplicasBelowMinDynClusterSize` | Boolean | Whether to allow the number of running cluster member Managed Server instances to drop below the minimum dynamic cluster size configured in the WebLogic domain configuration, if this is not specified for a specific cluster under the `clusters` field. Defaults to true. |
 | `clusters` | array of [Cluster](#cluster) | Lifecycle options for all of the Managed Server members of a WebLogic cluster, including Java options, environment variables, additional Pod content, and the ability to explicitly start, stop, or restart cluster members. The `clusterName` field of each entry must match a cluster that already exists in the WebLogic domain configuration. |
+| `commonMountVolumes` | array of [Common Mount Volume](#common-mount-volume) | The common mount tuning. Allows overriding the global default values for mountPath, medium and sizeLimits for the common mount. See spec.commonMountTuning.globalDefaults for more details. |
```

Comment:
I thought commonMountTuning is going away?  The description should be something like "Configure common mount volumes including their respective mount paths. Common mount volumes are in turn referenced by one or more `serverPod.commonMount` mounts, and are internally implemented using a Kubernetes 'emptyDir' volume."

Reply:
sorry, the current set of changes are a work in progress based on new schema, I still need to do code cleanup. I have converted the PR to be draft.